contrib/ci-workers/README.md

```diff
@@ -0,0 +1,42 @@
+# Zcash CI workers
+
+This folder contains the Ansible playbooks for configuring a fresh OS
+installation for use as a Buildbot worker in Zcash's CI.
+
+# Setting up a latent worker on Amazon EC2
+
+- Add a regular (non-latent) worker to the master.cfg for ci.z.cash, and deploy
+  the changes.
+  - This enables the Ansible playbook to run to completion, ending in the worker
+    connecting to the master.
```

Comment:
When we have a dev-ci set up (and have either figured out how to set up the workers for both, or extended this playbook to enable workers to be switched between dev-ci and prod-ci), we should use dev-ci for this step.